cas_server/views.py

```diff
@@ -626,6 +626,23 @@ def process_get(self):
         """
         # generate a new LT
         self.gen_lt()
+        if settings.CAS_DJANGO_AUTOLOGIN and self.request.user.is_authenticated:
+            self.request.session.set_expiry(0)
+            self.request.session["username"] = self.request.user.username
+            self.request.session["warn"] = False
+            self.request.session["authenticated"] = True
+            self.renewed = True
+            self.warned = True
```

Comment:
This should be false as well, and it won't do anything since via Django auth, we cannot known if the user want to be warn before been authenticated on a service or not.